jupyter-notebooks/gee-integration/gee-integration.ipynb

```diff
@@ -0,0 +1,324 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Order and Delivery to Google Earth Engine\n",
+    "\n",
+    "In this notebook we're going to cover how to order data using the [Orders API](https://developers.planet.com/apis/orders/) and deliver it to [Google Earth Engine (GEE)](https://earthengine.google.com/) using [Planet’s GEE Delivery integration](https://developers.planet.com/docs/integrations/gee/).\n",
+    "\n",
+    "This example demonstrates how to:\n",
+    "1. Define the data to be ordered\n",
+    "2. Build a cloud delivery configuration object (`delivery_config`), which tells the Orders API where to deliver the data\n",
+    "3. Build an order request to be sent off to the Orders API, `iowa_order`\n",
+    "4. Create the order and have it deliver to your GEE project\n",
+    "\n",
+    "**Prerequsites:** \n",
```

Comment:
Considering our discussion in slack it might be nice to advertise BYO service account here 

https://developers.planet.com/docs/integrations/gee/delivery/#bring-your-own-service-account

Reply:
This is something I was debating on adding to this notebook, but my team and I landed on making this its own notebook. I created a ticket for exactly this, #223, thanks for pointing this out.